### ISSUE TITLE
KERI event stream - detail KEL/TEL events for anchoring AlsoKnownAs, EquivalentID, and other DID metadata

### ISSUE THREAD

**Issue by 2byrds** [trustoverip/tswg-did-method-webs-specification#51]
We need a way to say how "AlsoKnowAs" values are injected into the KEL/TEL, included in the end states of those, and then put into the DIDDoc. The ones we care about are when a did:webs is redirected to a new URL by its controller, so that existing references to the "old" DID are not useless.

**Comment by peacekeeper** [comment]
Yes we need to talk a bit more about `alsoKnownAs` (see also https://github.com/trustoverip/tswg-did-method-webs-specification/issues/11), but in general I find it a bit difficult to ask "what KEL/TEL events to you need in order to create a certain property in the DID document". The `alsoKnownAs` property is a good case where this question is difficult. I imagine that there will ALWAYS be an `alsoKnownAs` that points to the did:keri equivalent of did:webs. There is no explicit event that you need to generate in order to anchor this `alsoKnownAs` entry. It's simply there because the spec says so. So I think the better question is to ask "how is the KEL/TEL mapped to a DID document", and from that it follows what kind of events result in what kind of changes in the DID document.

**Comment by swcurran** [comment]
I think we want to explicitly support having a “did:webs” that is (web-terms) redirected to another URL. That is the specific use case for an event related to `alsoKnownAs` — allowing the controller to say that this was an intended event.

I think that is important, although I think I could be convinced otherwise :-)

**Comment by 2byrds** [comment]
I need to add examples from the reference implementation of designated aliases

**Comment by 2byrds** [comment]
Fixed in https://github.com/trustoverip/tswg-did-method-webs-specification/pulls
